tests/integration/cattletest/core/test_docker.py

```diff
@@ -402,20 +413,20 @@ def test_docker_volumes(client, admin_client, docker_context):
     baz_host_path = '/tmp/baz%s' % bind_mount_uuid
     baz_bind_mount = '%s:/baz:ro' % baz_host_path
 
-    c = admin_client.create_container(name="volumes_test",
-                                      imageUuid=uuid,
-                                      startOnCreate=False,
-                                      dataVolumes=['/foo',
-                                                   bar_bind_mount,
-                                                   baz_bind_mount])
+    c = internal_test_client.create_container(name="volumes_test",
```

Comment:
admin_client should be used to create this container, perform actions on it, and assert the vast majority of fields.